stage1_fly/aci/aci-manifest.in

```diff
@@ -28,6 +28,14 @@
         {
             "name": "coreos.com/rkt/stage1/gc",
             "value": "/gc"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/stop",
+            "value": "/stop_fly"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/interface-version",
+            "value": "3"
```

Comment:
Before the patch, interface-version was not specified, so it is implicitely version=1, so it would print:

> stage0: warning: --hostname option is not supported by stage1

which is ok.

With the patch, I guess it would print (not fully tested):

```
$ sudo ./build-rkt-1.7.0+git/bin/rkt --debug  run --insecure-options=image --stage1-path=$PWD/build-rkt-1.7.0+git/bin/stage1-fly.aci docker://busybox --exec /bin/sh --hostname=river
...
stage0: Execing /run
flag provided but not defined: -hostname
Usage of /var/lib/rkt/pods/run/3aba2167-517d-42a2-bcf5-f0529e52443b/stage1/rootfs/run:
  -debug
    	Run in debug mode
  -interactive
    	The pod is interactive (default true)
  -local-config string
    	Local config path (default "/etc/rkt")
  -mds-token string
    	MDS auth token
  -net value
    	Setup networking (default default)
```

Which is less user-friendly, so I think that's a regression of this PR.